src/pages/generate/utils/analytics.ts

```
@@ -0,0 +1,423 @@
+ * - resultImg_click_moreModalMakeNew
+ * - resultImg_click_btnTag
+ * - resultImg_swipe_curationSheetUp
+ * - resultImg_swipe_curationSheetDown
+ * - resultImg_click_curationSheetFilterFurniture
+ * - resultImg_click_curationSheetBtnGoSite
+ * - resultImg_click_curationSheetBtnSave
+ * - resultImg_click_curationSheetCard
+ *
+ */
+
+import { logEvent } from 'firebase/analytics';
```

ESLint

[error] Replace `'firebase/analytics'` with `"firebase/analytics"`

(prettier/prettier)